src/SDKs/Batch/DataPlane/Azure.Batch.Unit.Tests/ComputeNodeUnitTests.cs

```diff
@@ -6,13 +6,17 @@
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using System.Threading.Tasks;
     using BatchTestCommon;
     using Microsoft.Azure.Batch;
     using Microsoft.Azure.Batch.Common;
+    using Microsoft.Azure.Batch.Auth;
+    using Microsoft.Rest.Azure;
     using TestUtilities;
     using Xunit;
     using Xunit.Abstractions;
     using Protocol=Microsoft.Azure.Batch.Protocol;
+    using Models = Microsoft.Azure.Batch.Protocol.Models;
```

Comment:
Slightly puzzled why we seem to be adding a bunch of using directives to this class while not making any other changes.  Did things move around?  Is GitHub showing a dodgy diff?